.github/workflows/test-python-package.yml

```
@@ -22,14 +22,17 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8
+        python -m pip install ruff flake8
         python -m pip install .[tests]
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 zoidberg --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 zoidberg --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with ruff
+      run: |
+        ruff check zoidberg
```

```suggestion
        ruff check zoidberg --output-format=github
```

This will automatically make annotations :)